holoviews/core/data/__init__.py

```diff
@@ -281,15 +279,15 @@ def range(self, dim, data_range=True, dimension_range=True):
 
         if dim is None or (not data_range and not dimension_range):
             return (None, None)
-        elif all(util.isfinite(v) for v in dim.range) and dimension_range:
+        elif all(isfinite(v) for v in dim.range) and dimension_range:
             return dim.range
         elif dim in self.dimensions() and data_range and len(self):
             lower, upper = self.interface.range(self, dim)
         else:
             lower, upper = (np.NaN, np.NaN)
         if not dimension_range:
             return lower, upper
-        return util.dimension_range(lower, upper, dim.range, dim.soft_range)
+        return d_range(lower, upper, dim.range, dim.soft_range)
```

Comment:
I think I prefer the old version...I don't see why the name `d_range` should be introduced for a single use (at least in this PR diff...)

Reply:
Again, see the comment above.

Reply:
Well, really it is two things:

1. For some reason you have problems with the qualified import (the root problem which would ideally be fixed)
2. You renamed to `d_range` to avoid a local variable name clash.

This PR would be greatly simplified if 1 can be fixed!

Reply:
Having nightmares with this, but I should be able to fix it.